src/lib/db/feature-toggle-client-store.ts

```diff
@@ -203,6 +222,20 @@ export default class FeatureToggleClientStore
         );
     }
 
+    private isUnseenTag(
+        feature: PartialDeep<IFeatureToggleClient>,
+        row: Record<string, any>,
+    ): boolean {
+        return (
+            row.tag_type &&
+            row.tag_value &&
+            !feature.tags.find(
```

Comment:
Probably better to use `!some` instead of returning object and converting to boolean.